api/docs.go

```diff
@@ -34,6 +34,50 @@ const docTemplate = `{
                 ],
                 "summary": "List Repositories",
                 "operationId": "listRepositories",
+                "parameters": [
+                    {
+                        "type": "integer",
+                        "description": "Offset into the list of results to return in the response",
+                        "name": "offset",
+                        "in": "query"
+                    },
+                    {
+                        "type": "integer",
+                        "description": "Limit the number of items returned",
+                        "name": "limit",
+                        "in": "query"
+                    },
+                    {
+                        "type": "string",
+                        "description": "Comma separated list of architecture to optionally filter-on (IE 'x86_64,s390x' would return Repositories with x86_64 or s390x only)",
```

Comment:
Not sure if this is used elsewhere, but `IE 'some,thing` isn't really appropriate here.

1. It needs to be an abbreviation, and shouldn't be capitalized (i.e.)
2. `e.g.` is more appropriate because its an example, and the latin translation is `for example`.

Reply:
updated